### PR TITLE
I18n translations for fnd 1431

### PR DESCRIPTION
**Pull request by JamesHankinsDev**
mall PR to add i18n strings for FND-1431: Lock Section

Adds required strings to React and Rails i18n files 
- `apps/i18n/common/en_us.json` implements for React, 
- `dashboard/config/locales/en.yml` implements for Rails

**NOTE: This sets up strings to be included in feature branch FND-1431: Section Lock and should be reviewed/approved/merged first.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -1169,6 +1172,8 @@
   "manageStudentsNotificationAtCapacity": "You can’t have more than 500 students in a section. Questions about this limit? [Contact support](https://code.org/contact)",
   "manageStudentsNotificationAddSuccess": "You added {numStudents, plural, one {1 student} other {# students}} to your section.",
   "manageStudentsNotificationSuccess": "Success!",
+  "manageStudentsNotificationLocked": "You have locked this section",
+  "manageStudentsNotificationLockedDetails": "Students can not join this section using the section code. {loginType, select, email {You can still add students to the section manually by moving or copying them from another section} other {You can still add students manually on the Manage Students tab}}. You can unlock this section by clicking \"Edit Section Details\".",
```
Comment (dju90) [inline apps/i18n/common/en_us.json]: Nit: "can not" should be one word: "cannot"

Comment (JamesHankinsDev) [reply apps/i18n/common/en_us.json]: Thank you! Just pushed a fix for that typo

**Review by dju90 — APPROVED** [review]

File: dashboard/config/locales/scripts.en.yml [hunk 2]
```diff
@@ -3980,7 +3980,7 @@ en:
               description_student: In this lesson the class develops a preliminary definition of a computer. After brainstorming the possible definitions for a computer, the class works in groups to sort pictures into “is a computer” or “is not a computer” on poster paper and explain their motivations for choosing some of the most difficult categorizations.  The teacher then introduces a definition of the computer and allows groups to revise their posters according to the new definition.
               description_teacher: 'In this lesson students develop a preliminary definition of a computer. To begin the lesson, the class will brainstorm possible definitions for a computer and place the results of this brainstorm on the board. Next, students will work in groups to sort pictures into “is a computer” or “is not a computer” on poster paper. Groups will place their posters around the room and briefly explain their motivations for choosing some of their most difficult categorizations.  The teacher will then introduce a definition of the computer and allow students to revise their posters according to the new definition.
 
-'
+                '
```
Comment (hacodeorg) [inline dashboard/config/locales/scripts.en.yml]: Why are there so many space changes in this file?

**Review by hacodeorg — COMMENTED** [review]
You don't have to check in the changes in `dashboard/db/schema_cache.yml`. 
It is taken care by the CI/CD pipeline https://github.com/code-dot-org/code-dot-org/commits/3ffbc6d787350a7e6b0b41d2000af16456adc3d0/dashboard/db/schema_cache.yml

File: dashboard/config/locales/scripts.en.yml (hunk 2 above)
Comment (JamesHankinsDev) [reply dashboard/config/locales/scripts.en.yml]: Sorry about that - just pulled the `scripts.en.yml` file out

**Review by hacodeorg — APPROVED** [review]

**Pull request merged**